### DISCUSSION TITLE
Method groups for instance members

### DISCUSSION THREAD

**Discussion by chylex** [dotnet/csharplang#5608] [Language Ideas] [answered]
Hi, I'd like to ask about a possibility of passing instance methods into anything that takes method groups. For example, instead of:

```csharp
File.ReadAllLines("file.txt").Select(static line => line.TrimEnd())
```

you could write:

```csharp
File.ReadAllLines("file.txt").Select(string::TrimEnd)
```

My proposal example uses `::` to disambiguate static and instance members, but it could conflict with the `::` [operator](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/namespace-alias-qualifier).

I highly suspect this has been proposed in the past, but I have searched both issues and discussions, and haven't been able to find any. If anyone knows of an existing proposal, please let me know.

**Comment by HaloFour** [comment, marked as answer]
See #84

**Reply by chylex** [reply]
Thanks! Turns out I'd already upvoted it a few years ago :) could be good to make the title more searchable with C# lingo, since from all the searches I tried the only commonality with the current title "Method References" was the keyword "Method".
